chapters/substitution.adoc

```diff
@@ -2,13 +2,22 @@
 .Definition
 Robots are substituted by the <<Robot Handler, robot handler>> of the respective team. No other team member is allowed to take robots out or put robots in.
 
-. A <<Robot Handler, robot handler>> who's team is marked for robot substitution performs at least one of the following operations in the given order.
-.. Put robots in. The maximum allowed number of robots of the team on the field must not be exceeded.
-.. Take robots out. If a robot that is taken out is closer than 1 meter to the intersection of the halfway line with one of the touch lines, another robot can be put in right away. Otherwise the robot handler has to wait until the next substitution opportunity.
+. Robot substitution during game play:
+.. The <<Robot Handler, robot handler>> should prefer to use long sleeves and colors that won't interfere with the vision system.
+.. A <<Robot Handler, robot handler>> who's team is marked for robot substitution performs at least one of the following operations in the given order.
+... Put robots in. If the ball is at least 2 meters away from the halfway line, the robot can be placed on the intersection of the halfway line with any touch line without notifying the <<Referee, referee>>.
+... Take robots out. If a robot is at the intersection of the halfway line with any touch line and the ball is at least 2 meters away from the halfway line, the robot can be removed without notifying the <<Referee, referee>>.
+
+. Robot substitution on next stoppage:
+.. A <<Robot Handler, robot handler>> who's team is marked for robot substitution performs at least one of the following operations in the given order.
+... Put robots in. The maximum allowed number of robots of the team on the field must not be exceeded.
```

Comment:
This is a bit of a nitpick, but if the robot handler follows these steps in order they may temporarily exceed the maximum number of robots of the team on the field because one must be added before another can be removed. Perhaps we should clarify the maximum allowed number of robots of the team on the field must not be exceeded _when the substitutions are finished_? Unless this is intended.

Reply:
I don't think this is intended, I believe that what it means is that the robot handler can perform any of those actions, not necessarily in order. Doesn't make much sense only being able to take a robot out if you put one in. It is the same idea as previous years, there can never be more robots than the allowed for each team, and, there's nothing that prohibits having less robots. I'll fix this.